website/content/en/docs/AWS/launch-templates.md

```diff
@@ -41,6 +41,20 @@ AMI (Amazon Machine Image), is the base image/VM for a launch template.
 [Review the instructions for importing a VM to AWS.](https://docs.aws.amazon.com/vm-import/latest/userguide/vmimport-image-import.html) Note the AMI id generated by this process, such as,
 `ami-074cce78125f09d61`.
 
+### Instance Profile - IAM
+
+The launch template must include an "instance profile" -- an IAM role.
+
+The instance profile must include *at least* the permissions of the default Karpenter node instance profile. See the default role, `KarpenterNodeRole`, in the full example below for more information.
+
+See also, [the managed policy "AmazonEKSWorkerNodePolicy"](https://docs.aws.amazon.com/eks/latest/userguide/security-iam-awsmanpol.html#security-iam-awsmanpol-AmazonEKSWorkerNodePolicy) which includes permission to describe clusters and subnets.
+
+### Tags
+
+Instances must have the tag `kubernetes.io/cluster/EksClusterName` and value `owned` to successfully join the EKS cluster. 
+
+Note that launch templates have two resource tag sections, one for the launch template itself, and another for the instances it creates. Check that the instances are tagged. 

```

Comment:
We actually override the resource tag w/ fleet overrides, so they don't need to be specified.

Reply:
i'm confused. if karpenter overrides these tags, why does not including them in the LT cause nodes to fail to join?

Reply:
This is surprising! It's possible we're not adding all of the right ones? @bwagner5 do you have any idea?

Reply:
Implemented here https://github.com/aws/karpenter/commit/9af7af01f94a1a172a38ad404c8158153f488bd7#diff-bc42bdb45b943498089740b1cc573ac30b5cfc994dc76981636daea88ae1d495